src/lib/libraries/sprites.json

```diff
@@ -14328,7 +14328,7 @@
                 {
                     "costumeName": "text keep scratching",
                     "baseLayerID": -1,
-                    "baseLayerMD5": "22a916248a294eadc07a7a065f5154e8.svg",
+                    "baseLayerMD5": "b3de9a77e2c1bb2f3e29f29a03d869ea.svg",
```

Comment:
Please update to `97c9e6ef78d4f1987fd8c6e5042963cb.svg` and it will start working 🌈